src/runtime.js

```diff
@@ -179,10 +179,11 @@ function adoptShared(global, opts, callback) {
   if (!global.AMP.extension) {
```

Comment:
Can we go ahead and replace this `if` with `minified` or whatever `if` that will strip this function from PROD version?